src/main/java/com/health/controller/AjaxController.java

```diff
@@ -883,6 +883,24 @@ private HashMap<String, String> setResponse(Integer status) {
 
     }
 
+    @RequestMapping("/loadTopicByCategoryforCitation")
+    public @ResponseBody HashMap<Integer, String> getTopicByCategoryforCitation(@RequestParam(value = "id") int id) {
+        HashMap<Integer, String> topicName = new HashMap<>();
+
+        Category cat = catService.findByid(id);
+        logger.info("Variables of getTopicByCategory catId :{} category : {}", id, cat);
+
+        List<Tutorial> tutorials = tutService.findAllEnabledEnglishTuorialsWithCitationIsNull();
```

Comment:
```suggestion
        List<Tutorial> tutorials = tutService.findAllEnabledEnglishTuorialsWithCitationIsNull(cat);
```
Search by cat here itself (change function name also, if required)